users/details/style.css

```diff
@@ -140,6 +140,25 @@ li {
   height: 25px;
 }
 
+.enabled {
+  background-color: var(--loader-gray-color);
+  color: black;
+  cursor: pointer;
+}
+
+.disabled {
+  position: relative;
+  background-color: #dddddd;
+  color: #999999;
```

Comment:
Same as above create variable for the colors

Reply:
why everything works fine in ui part I found no issue

Reply:
made necessary changes in latest commit